cmd/oceanbench/main.go

```diff
@@ -352,6 +352,44 @@ func setup(ctx context.Context) {
 	}
 
 	iotds.RegisterEntities()
+
+	err = notifier.Init(ctx, projectID, senderEmail, &timeStore{})
+	if err != nil {
+		log.Fatalf("could not set up email notifier: %v", err)
+	}
+
+	const templateDir = "t"
```

Comment:
You've picked up these template changes from a previously discarded branch. Please re-pull from main.